### PR TITLE
Default retry

### PR DESCRIPTION
**Pull request by m1racoli**
Default retry parameters changed for:
- SqlSensor
- ExasolOperator

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
         'flask-cache>=0.13.1',
         'flask-login>=0.2.11',
         'flower>=0.7.3',
+        'jaydebeapi>=0.2.0',
```
Comment (mistercrunch) [inline setup.py:43]: Can you switch to using the `extra_requires` to make it optional? You can call it `jdbc`. What this does is that if you want to install this extra, you have to write something like `pip install airflow[jdbc]`

**Comment by mistercrunch** [comment]
Very nice. This opens all of the JDBC doors. Would it make sense to generalize and simply call the hook JdbcHook? The operator JdbcOperator? It would have to receive the driver information as a param but I think that's all it would take...

**Comment by m1racoli** [comment]
Sure, I'll close this and make a new PR when I have made the changes. I got an additional commit in this one, which I rather take out again, since it's a custom change for our operations.

**Comment by mistercrunch** [comment]
Sure thing. I'm excited on how this will make all of JDBC compatible with Airflow!

**Pull request closed without merging**